web/skins/classic/views/options.php

```diff
@@ -467,6 +467,7 @@ function format_timezone_name($name) {
   foreach ($configCat as $name=>$value) {
     $shortName = preg_replace('/^ZM_/', '', $name);
     $optionPromptText = !empty($OLANG[$shortName])?$OLANG[$shortName]['Prompt']:$value['Prompt'];
+    $optionPromptText = translate(optionPromptText);
```

Comment:
I think we are missing a $ here on optionPrompText

Also... OLANG is already translated I think.  So it's like a double translate.

Reply:
Hey Issac you're right about the missing $, sorry still working on github skills (did the edit locally and then change it on repo manually)... 

About the $OLANG, it doesn't translate without that line, at least using language files, but if you want I can check why is not translating.